File: content-solutions/hash/class-vote.js
function solution(votes) {
  const count = new Map();
  for (const v of votes) {
    count.set(v, (count.get(v) ?? 0) + 1);
  }
  let best = null;
  for (const [name, c] of count) {
    if (best === null || c > count.get(best) || (c === count.get(best) && name < best)) {
      best = name;
    }
  }
  return best;
}

File: content-solutions/hash/class-vote.py
def solution(votes):
    count = {}
    for v in votes:
        count[v] = count.get(v, 0) + 1
    best = None
    for name, c in count.items():
        if best is None or c > count[best] or (c == count[best] and name < best):
            best = name
    return best
